dora/stdlib/Float32.dora

```diff
@@ -33,6 +33,7 @@ impl Float32 {
   @pub @internal fun unaryPlus(): Float32;
   @pub @internal fun unaryMinus(): Float32;
   @pub @internal fun abs(): Float32;
+  @pub @internal fun roundHalfEven(): Float32;
```

Comment:
I'm currently planning on having a method for each of the ~four rounding modes.

Alternative design is having one `fun round(RoundingMode)` method, but I first like to wait and see in how many additional places we would use such an enum.